### PR TITLE
Deprecate bridge peripherals

### PR DESCRIPTION
**Pull request by Jason-Wanxt**

### CODE REVIEW

**Review by gzeoneth — DISMISSED** [review]
LGTM

**Review by fredlacs — CHANGES_REQUESTED** [review]
lets do the same to `arb-bridge-eth`

File: packages/arb-bridge-peripherals/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# arb-bridge-peripherals
+
+This package has deprecated!
```
Comment (fionnachan) [inline packages/arb-bridge-peripherals/README.md:3]:
```suggestion
This package has been deprecated!
```

**Review by fredlacs — COMMENTED** [review]
lets add a note in `arb-bridge-eth`'s README

**Review by gzeoneth — DISMISSED** [review]
one more change

File: packages/arb-bridge-eth/README.md [hunk 2]
```diff
@@ -8,3 +8,9 @@ sudo docker run -p 7545:7545 -it arb-bridge-eth
 ```
 
 This repository is offered under the Apache 2.0 license. See LICENSE for details.
+
+## Only for classic version
+
+This package has been deprecated!
+
+If you want to use what was once called `arb-bridge-eth`, look into the latest and greatest [`token-bridge-contracts`](https://github.com/OffchainLabs/token-bridge-contracts)!
```
Comment (gzeoneth) [inline packages/arb-bridge-eth/README.md:16]: should be nitro contracts instead, this is the rollup bridge

**Review by fredlacs — APPROVED** [review]
LGTM

**Review by fredlacs — APPROVED** [review]
LGTM

**Pull request merged**